app/frontend/src/components/Bar/ScoreBar.tsx

```diff
@@ -38,8 +38,10 @@ interface ScoreBarProps extends ContainerProps {
 
 export default function SearchResult({ value, children }: ScoreBarProps) {
   const classes = useStyles();
-  return (
+  return process.env.REACT_APP_IS_BETA_ENABLED ? null : (
     <Container disableGutters className={classes.root}>
+      {`hello: ${process.env.REACT_APP_MAPBOX_KEY}`}
+      {/* {`bye: ${process.env.REACT_APP_IS_BETA_ENABLED}`} */}
```

Comment:
```suggestion
      {`hello: ${process.env.REACT_APP_MAPBOX_KEY}`}
      {/* {`bye: ${process.env.REACT_APP_IS_BETA_ENABLED}`} */}
```
I'm guessing this is not meant to be left in? 😄